### PR TITLE
launches web3 what / why video

### PR DESCRIPTION
**Pull request by owocki**
https://www.youtube.com/watch?v=cZZMDOrIo2k

takes this video (which is final now) and launches it across the site.

### CODE REVIEW

**Comment by owocki** [comment]
@vs77bb let me know if we want to time this release to any content piece... 

at the very least we might want to put it in the gitcoin.co/onboard doc

File: app/assets/legacy/v2/js/shared.js [hunk 1]
```diff
@@ -440,7 +440,7 @@ window.addEventListener('load', function() {
         if (typeof web3 =='undefined'){
             $("#upper_left").addClass('disabled');
             $("#sidebar_head").html("Web3 disabled <br> <img src='/static/v2/images/icons/question.png'>");
-            $("#sidebar_p").html("Please install <a target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://metamask.io/?utm_source=gitcoin.co&utm_medium=referral\">Metamask</a> <br> <a target=new href='https://github.com/gitcoinco/gitcoinco/issues/4'>What is Metamask and why do I need it?</a>.");
+            $("#sidebar_p").html("Please install <a target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://metamask.io/?utm_source=gitcoin.co&utm_medium=referral\">Metamask</a> <br> <a target=new href='/web3'>What is Metamask and why do I need it?</a>.");
```
Comment (mbeacom) [inline app/assets/legacy/v2/js/shared.js]: Can we modify this to use url reverse lookup?

File: app/assets/v2/js/shared.js [hunk 2]
```diff
@@ -463,7 +463,7 @@ window.addEventListener('load', function() {
         if (typeof web3 =='undefined'){
             $("#upper_left").addClass('disabled');
             $("#sidebar_head").html("Web3 disabled <br> <img src='/static/v2/images/icons/question.png'>");
-            $("#sidebar_p").html("Please install <a target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://metamask.io/?utm_source=gitcoin.co&utm_medium=referral\">Metamask</a> <br> <a target=new href='https://github.com/gitcoinco/gitcoinco/issues/4'>What is Metamask and why do I need it?</a>.");
+            $("#sidebar_p").html("Please install <a target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://metamask.io/?utm_source=gitcoin.co&utm_medium=referral\">Metamask</a> <br> <a target=new href='/web3'>What is Metamask and why do I need it?</a>.");
```
Comment (mbeacom) [inline app/assets/v2/js/shared.js]: Same

File: app/dashboard/templates/shared/metamask_estimate.html [hunk 3]
```diff
@@ -35,7 +35,7 @@
                                     <strong>Help:</strong>
                                     <ul>
                                         <li>
-                                            <a target=new href="https://github.com/gitcoinco/gitcoinco/issues/4">Why do I need metamask / gas?</a>
+                                            <a target=new href="/web3">Why do I need metamask / gas?</a>
```
Comment (mbeacom) [inline app/dashboard/templates/shared/metamask_estimate.html]: Same

File: app/retail/views.py [hunk 4]
```diff
@@ -266,7 +266,7 @@ def help(request):
 <p>Gitcoin is similarly built on an open protocol of smart contracts.</p>
 <p>By specifying a&nbsp;protocol, Tim Berners-Lee opened the way for anyone to build software, so-called web servers and browsers that would be compatible with this protocol. &nbsp; By specifying an open source protocol for Funding Issues and software development scoping &amp; payment, the Gitcoin Core team hopes to similarly inspire a generation of inventions in 21st century software.</p>
 <p>
-To learn more about blockchain, please checkout the <a href="https://github.com/gitcoinco/gitcoinco/issues?q=is%3Aissue+is%3Aopen+label%3Ahelp">Github Issues board</a>
+To learn more about blockchain, please checkout <a href="/web3">this video about web3</a> or the <a href="https://github.com/gitcoinco/gitcoinco/issues?q=is%3Aissue+is%3Aopen+label%3Ahelp">Github Issues board</a>
```
Comment (mbeacom) [inline app/retail/views.py]: Same

File: app/retail/views.py [hunk 5]
```diff
@@ -289,7 +289,7 @@ def help(request):
 Download Metamask <a href="https://metamask.io/">here</a> today.
 </p>
 <p>
-To learn more about Metamask, please checkout the <a href="https://github.com/gitcoinco/gitcoinco/issues?q=is%3Aissue+is%3Aopen+label%3Ahelp">Github Issues board</a>
+To learn more about Metamask, please checkout <a href="/web3">this video about web3</a> or the <a href="https://github.com/gitcoinco/gitcoinco/issues?q=is%3Aissue+is%3Aopen+label%3Ahelp">Github Issues board</a>
```
Comment (mbeacom) [inline app/retail/views.py]: Same

File: app/retail/views.py [hunk 6]
```diff
@@ -333,7 +333,7 @@ def help(request):
 </li>
 </ul>
 <p>
-To learn more about Ethereum based apps, please checkout the <a href="https://github.com/gitcoinco/gitcoinco/issues?q=is%3Aissue+is%3Aopen+label%3Ahelp">Github Issues board</a>
+To learn more about Ethereum based apps, please checkout <a href="/web3">this video about web3</a> or the <a href="https://github.com/gitcoinco/gitcoinco/issues?q=is%3Aissue+is%3Aopen+label%3Ahelp">Github Issues board</a>
```
Comment (mbeacom) [inline app/retail/views.py]: Same

File: app/app/urls.py [hunk 7]
```diff
@@ -123,6 +123,7 @@
     url(r'^medium/?', retail.views.medium, name='medium'),
     url(r'^github/?', retail.views.github, name='github'),
     url(r'^youtube/?', retail.views.youtube, name='youtube'),
+    url(r'^web3/?', retail.views.web3, name='web3'),
```
Comment (mbeacom) [inline app/app/urls.py]: Can we move forward with `path` and `re_path`?

File: app/assets/legacy/v2/js/shared.js (hunk 1 above)
Comment (owocki) [reply app/assets/legacy/v2/js/shared.js]: its in the JS layer... not sure how we'd do that there.

File: app/assets/v2/js/shared.js (hunk 2 above)
Comment (owocki) [reply app/assets/v2/js/shared.js]: its in the JS layer... not sure how we'd do that there.

File: app/dashboard/templates/shared/metamask_estimate.html (hunk 3 above)
Comment (owocki) [reply app/dashboard/templates/shared/metamask_estimate.html]: updated

File: app/retail/views.py (hunk 4 above)
Comment (owocki) [reply app/retail/views.py]: updated

File: app/retail/views.py (hunk 5 above)
Comment (owocki) [reply app/retail/views.py]: updated

File: app/retail/views.py (hunk 6 above)
Comment (owocki) [reply app/retail/views.py]: updated

File: app/assets/legacy/v2/js/shared.js (hunk 1 above)
Comment (mbeacom) [reply app/assets/legacy/v2/js/shared.js]: Whoops

**Pull request merged**